packages/react/src/components/Slider/useSlider.ts

```diff
@@ -13,6 +13,8 @@ import {
 } from '@fluentui/utilities';
 
 export const ONKEYDOWN_TIMEOUT_DURATION = 1000;
+const MIN_PREFIX = 'min';
+const MAX_PREFIX = 'max';
```

Comment:
You don't need these; just use 'min'/'max' inline in the two places each one is used.